### PR TITLE
Ask people to tell us if they use our standards elsewhere

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

**Comment by noamross** [comment]
@stefaniebutland What is the best way for people who may use our guidelines on other projects / reviewing in other venues to let us know about it?

**Comment by stefaniebutland** [comment]
I think the forum Use Cases category is the right place https://discuss.ropensci.org/c/usecases

We created a template to guide people. In part, it says:
> #### Package or resource used*
>_e.g. one of our [packages](https://ropensci.org/packages/), our [package development guide book](https://ropensci.github.io/dev_guide/), a [Community Call](https://ropensci.org/commcalls/), or a [blog post](https://ropensci.org/blog/)_

But I could make this more explicit in the category description and template (whose opening text refers more to packages). What do you think? I've been pondering a blog post that would encourage people to share there.

I could ask Hao Ye to add his use our software review guidelines there. 

Aside: fyi, I have been encouraging people to post their use cases to the forum, and I schedule a tweet about each one for Wednesdays.

**Comment by maelle** [comment]
So we could direct them to the forum, and what alternative if they do not want to write publicly e.g if the guidelines were refused?

**Comment by noamross** [comment]
I agree, the forum makes sense in general, along with "or drop us a line at ...." with an email address.

**Comment by stefaniebutland** [comment]
@maelle I'll include updated text to address this under "non-code contributions" in my draft text revision for "14.1 Why contributing to rOpenSci packages?"

Planning to send to you first as google doc, by Mon May 4 latest

File: softwarereview_intro.Rmd [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 ```{block, type='summaryblock'}
 This chapter contains a [general intro](#whatissoftwarereview) to our software peer review system for packages, [reasons to submit a package](#whysubmit), [reasons to volunteer as a reviewer](#whyreview), [why our reviews are open](#whyopen), and acknowledgements of [actors of the system](#editors-and-reviewers).
+
+_If you use our standards/checklists etc. when reviewing software elsewhere, do tell your editors where they came from, and [tell us](https://ropensci.org/contact/) if it's allowed._
```
Comment (stefaniebutland) [inline softwarereview_intro.Rmd]:
```suggestion
_If you use our standards/checklists etc. when reviewing software elsewhere, do tell the recipients (e.g. journal editors, students, internal code review) that they came from rOpenSci, and tell us in [our public forum](https://discuss.ropensci.org/c/usecases), or [privately by email](https://ropensci.org/contact/)._ 
```

File: softwarereview_reviewer.Rmd [hunk 2]
```diff
@@ -7,6 +7,8 @@ You might contact the editor in charge of the submission for any question you mi
 
 Please strive to complete your review within 3 weeks of accepting a review request. We will aim to remind reviewers of upcoming and past due dates. Editors may assign
 additional or alternate reviewers if a review is excessively late.
+
+_If you use our standards/checklists etc. when reviewing software elsewhere, do tell your editors where they came from, and [tell us](https://ropensci.org/contact/) if it's allowed._
```
Comment (stefaniebutland) [inline softwarereview_reviewer.Rmd]:
```suggestion
_If you use our standards/checklists etc. when reviewing software elsewhere, do tell the recipients (e.g. journal editors, students, internal code review) that they came from rOpenSci, and tell us in [our public forum](https://discuss.ropensci.org/c/usecases), or [privately by email](https://ropensci.org/contact/)._ 
```

**Comment by maelle** [comment]
Thank you!

**Pull request merged**